pkg/apis/go.mod

```diff
@@ -29,16 +27,16 @@ replace (
 )
 
 require (
-	github.com/rancher/aks-operator v1.3.0-rc1
-	github.com/rancher/eks-operator v1.4.0-rc1
+	github.com/rancher/aks-operator v1.3.0-rc1.0.20240124091359-b8e30207d302
+	github.com/rancher/eks-operator v1.4.0-rc1.0.20240123182902-b173c90d7965
 	github.com/rancher/fleet/pkg/apis v0.0.0-20231017140638-93432f288e79
-	github.com/rancher/gke-operator v1.3.0-rc2
-	github.com/rancher/norman v0.0.0-20240205154641-a6a6cf569608
+	github.com/rancher/gke-operator v1.3.0-rc2.0.20240123183119-1ade4427ef28
```

Comment:
No familiar with this notation for a version. Can you elaborate on it maybe just for my curiosity?

Reply:
This was the version go.mod seems to select for the head head of a specific branch.

Reply:
Ok, it doesn't do that for tag afaik so that'd mean these -rcs are branches instead? Weird, anyway, that's fine thanks!